src/dashboard/messages.ts

```diff
@@ -0,0 +1,14 @@
+export default {
+    REGISTRATION_CANCELLED:                   'The setup process has been aborted.',
+    REGISTRATION_ENTER_EMAIL_INVITATION:      'Log into dashboard.testcafe.io to start the setup process.\nEnter your email to receive a login link.',
+    REGISTRATION_EMAIL_SENDING_NETWORK_ERROR: 'Error: cannot request the log-in email. Please check your internet connection.',
+    REGISTRATION_EMAIL_SENT:                  'Check your inbox for the TestCafe Dashboard log-in email. Follow the enclosed link to accept the EULA and log into the application.\nCopy the project token and paste it here.',
+    PROMPT_EMAIL_CAPTION:                     'Email:',
+    PROMPT_TOKEN_CAPTION:                     'Enter the project token:',
+    PROMPT_INVALID_EMAIL:                     'Please enter a valid email address',
+    TOKEN_VALIDATION_NETWORK_ERROR:           'Error: cannot validate the project token. Please check your internet connection.',
+    TOKEN_UPDATE_CANCELLED:                   'The token update has been aborted.',
+    TOKEN_UPDATED:                            'The default token has been updated.',
+    SEND_REPORT_STATE_UPDATED:                'The report sending state has been updated.',
```

Comment:
this message is useless because it doesn't indicate the current state of the dashboard reporter. Can we make it unique for each of the scenarios?

```suggestion
    SEND_REPORT_STATE_ON:                'TestCafe Dashboard integration enabled. From now on, this TestCafe installation will send test report data to TestCafe Dashboard.',
    SEND_REPORT_STATE_OFF:                'TestCafe Dashboard integration disabled. From now on, this TestCafe installation will no longer send test report data to TestCafe Dashboard.',
```

if not, the message below will do just fine, because it's equally informative, but not confusing:
```suggestion
    SEND_REPORT_STATE_UPDATED:                'Done.',
```

Reply:
Yes, we can use unique messages.